File: Entities/EskeysEntities/EskeyReceivable.cs
namespace PMS.Entities.EskeysEntities
{
    public class EskeyReceivable
    {
        public int Id { get; set; }
        public int Count { get; set; }
        public string location { get; set; } = String.Empty;
        public DateTime DateProcessed { get; set; }
        public DateTime ArrivalDateTime { get; set; }
        public DateTime? DateTimeReceived { get; set; }
        public int OperatorId { get; set; }
        public string Charter { get; set; } = String.Empty;
        public AppUser? Operator { get; set; }
        public List<EskeyReceivableDetail> EskeyReceivableDetails { get; set; } = new List<EskeyReceivableDetail>();
    }
}

File: ClientApp/src/app/models/receivablesAgg/eskeyReceivable.ts
import { EskeyReceivableDetail } from './eskeyReceivableDetail';
export interface EskeyReceivable {
  id: number;
  count: number;
  location: string;
  arrivalDateTime: string;
  operatorId: number;
  dateProcessed: string;
  dateTimeReceived: string;
  charter: string;
  eskeyReceivableDetails: EskeyReceivableDetail[];
}
